src/renderer/components/dock/log.store.ts

```diff
@@ -146,6 +146,16 @@ export class LogStore {
     return stamp.toISOString();
   }
 
+  splitOutTimestamp(logs: string): [string, string] {
+    const extraction = /^(\d+\S+) (.*)/gm.exec(logs);
```

Comment:
I don't think this needs to be global. Also wouldn't I don't think that space should be there between the two groups.

Reply:
But if I don't match the space between the two groups it will end up in my second capture group, which would require me to trim the string afterward

Reply:
Wouldn't that be a better solution? Since their might be spaces before hand? Though TBH #2412 implies that we shouldn't be trimming at all.

Reply:
Ok, so I've removed the space in the regex and in consequence of that I've also removed the space in the log formatter (https://github.com/lensapp/lens/pull/2523/commits/dbc44baea28d982d96ba378409ae62a76b0b15ca#diff-92bc22dec3f8bbca956be35e96f4152145e3b798349bf5ae04d5cf38e94cc854R92) so that I won't get an extra space.
However I'm not sure if that will fix #2412 as I'm just displaying what the kube API returns me

Reply:
Thanks, yeah I don't know if that will fix that entirely, but it might.